Jenkinsfile

```diff
@@ -108,6 +108,17 @@ pipeline {
             }
         }
 
+        stage("Integration Tests"){
+            when {
+                expression {
+                    return env.shouldBuild != "false"
+                }
+            }
+            steps {
+                sh './build/run ./cluster/local/integration_tests.sh'
```

Comment:
have you considered making this part of the make file targets? Something like `make test-integration`?

Reply:
I can do that. I was following the pattern in `cluster/local/minikube.sh`

Reply:
If it is not made a make file target I would suggest another directory due to the possible confusion with the other local development scripts that provide specific commands for developing Crossplane.